App.js

```diff
@@ -27,19 +39,23 @@ export default App;
 const styles = StyleSheet.create({
   appContainer: {
     flex: 1,
-    backgroundColor: '#F8EDDF',
+    backgroundColor: 'none',
     alignItems: 'center',
     // justifyContent: 'center',
   },
   image: {
     flex: 1,
+    height: '100%',
+    width: "100%",
+
   },
   bottomNavContainer: {
     // alignItems: 'flex-end',
     marginTop: -90,
     // justifyContent: 'flex-end',
     // height: 180,
-    width: "100%"
+    width: "100%",
+    margin: 10,
```

Comment:
Adding a margin here allowed the bottom navBar edges to meet the edge of the screen. Curious to see the display on larger screens.